### ISSUE TITLE
[FEATURE_REQUEST] Choose status dot position

### ISSUE THREAD

**Issue by alexdelprete** [Lissy93/dashy#583]
### Is your feature request related to a problem? If so, please describe.

The colored dot that presents the state of the service is on the upper right corner, next to the name of the service. With long titles it covers some letters.

![image](https://user-images.githubusercontent.com/7027842/160671421-f00928d3-6668-42a3-aca4-83b3f9c3b725.png)

### Describe the solution you'd like

An option (global or for each service) to choose the position of the dot (lower-right, lower-left, etc.)

I'd love to help, but I'm not a developer. :)

### Priority

Low (Nice-to-have)

**Comment by Lissy93** [comment]
You should be able to change this with CSS, if your unsure how I can send you a snippet when I get back to my PC this evening

**Comment by alexdelprete** [comment]
You're amazingly fast, there was no urgency. :)

I'll try to do it by myself, in case I will ask for your help, is there a single CSS file?

**Comment by alexdelprete** [comment]
I guess I'll wait for your help...don't know what to look for. :(

No urgency at all...low-priority.

**Comment by Lissy93** [comment]
Yup, that's the correct place to put any custom CSS. But you can also put it in your config file, under `appConfig` --> `customCss`, which is probably better, as it'll definitely stay backed up.

Here are some CSS snippets which will force the status indicator into different positions:
- Top-Left: `.indicator { left: 0; }`
- Top-Right: `.indicator { top: 0; right: 0; }`
- Bottom-Left: `.indicator { bottom: 0; left: 0; top: unset; }`
- Bottom-Right: `.indicator { bottom: 0; right: 0; top: unset; }`

If you need to set different positions for just certain items it's a bit more complicated. You'll need to find the ID for that item (visible in the edit item menu), then you can target it specifically with: `#link-1_1302_pihole .indicator { bottom: 0; right: 0; top: unset; }` (replace with actual ID, keep the #). Although moving an item can cause it's ID to change.

And if you need to force-override a CSS property, you can use `!important` before the semi colon, but use sparingly or things can get messy.

**Comment by Lissy93** [comment]
In the future, I will consider adding this as a config option
But have [a few other things](https://github.com/Lissy93/dashy/discussions/405) to get done first
If/ when it gets done, I will update you here :)

Feel free to reach out if there's anything else

**Comment by alexdelprete** [comment]
> Yup, that's the correct place to put any custom CSS. But you can also put it in your config file, under `appConfig` --> `customCss`, which is probably better, as it'll definitely stay backed up.

It didn't work when I edited the config file directly, don't know why. In the end, I used the integrated "Edit App Config" in Edit Mode:

![image](https://user-images.githubusercontent.com/7027842/160711014-83d512aa-09fb-4385-a5c7-566418062ca6.png)

I saved to disk and it updated the config file too. Refreshed and it worked beautifully:

![image](https://user-images.githubusercontent.com/7027842/160711230-dc4e536f-9721-4676-8d40-c34528597c2d.png)

Thank you Lissy, you're awesome, like Dashy. :)
